examples/react/urql-string-document-mode/src/App.tsx

```diff
@@ -0,0 +1,62 @@
+import { TypedDocumentString } from '@graphql-typed-document-node/core';
+
+import './App.css';
+import Film from './Film';
+import { graphql } from './gql';
+import { AnyVariables, OperationContext, RequestPolicy, TypedDocumentNode, useQuery, UseQueryResponse } from 'urql';
+import type { DocumentNode } from 'graphql';
+
+const allFilmsWithVariablesQueryDocument = graphql(/* GraphQL */ `
+  query allFilmsWithVariablesQuery($first: Int!) {
+    allFilms(first: $first) {
+      edges {
+        node {
+          ...FilmItem
+        }
+      }
+    }
+  }
+`);
+
+declare module 'urql' {
+  // @ts-expect-error this is just temporary until we update types in urql
```

Comment:
So I guess we should make a PR to `urql` after `@graphql-typed-document-node/core` is released, but what other places should we update?